etcdmain/etcd.go

```diff
@@ -379,3 +387,11 @@ func genClusterString(name string, urls types.URLs) string {
 	}
 	return strings.Join(addrs, ",")
 }
+
+// handleInterrupt calls the handler on receiving a SIGTERM.
+func handleInterrupt(h func()) {
+	notifier := make(chan os.Signal)
+	signal.Notify(notifier, os.Interrupt, syscall.SIGTERM)
```

Comment:
Oh yeah, you're right. My eyes skipped right over that. `syscall.SIGINT` is more clear. Thanks.